### ISSUE TITLE
Order Skip / Removals Service

### ISSUE THREAD

**Issue by scraphack** [cheltenhamhackspace/the_space#15]
We need a way to get rid of all the cruft, that will come in either the form of a skip or a removals service which will take general rubbish.

**Comment by ThePhilStrongProject** [comment]
Prerequisite is to give people the opportunity to retrieve their stuff, unless we have the skip long enough that they can throw their own junk out?

**Comment by scraphack** [comment]
Yep totally, comms need to go out about people retrieving their stuff to make space, anything left (after X amount of time) will need processing which will potentially lead to it being skipped.
